src/evrythng/tests/test_entities.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+import os
+from evrythng.entities import actions
+from evrythng.entities import products
+
+API_KEY = os.environ['API_KEY']
+
+
+class TestProductMethods(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        """Product Tests Setup"""
+
+        cls.product_id = products.create_product(
+            name='test_creating_prod_GooeeIOT', api_key=API_KEY).json()['id']
+
+    @classmethod
+    def tearDownClass(cls):
+        """Product Tests Destroying"""
+
+        products.delete_product(
+            product_id=str(cls.product_id), api_key=API_KEY)
+
+    def test1_creation(self):
```

Comment:
I personally don't like numbering tests like test1 test2 but thats just naming, overall looks good to me.